go.mod

```diff
@@ -3,11 +3,12 @@ module github.com/go-pkgz/lcw
 go 1.15
```

Comment:
ci switched to 1.18, but this one still 1.15. I suggest bumping to 1.18 as well

Reply:
I'll change this one alongside the generics introduction. Now the minimum because of `%w, err` formatting is 1.14.